packages/synapse-react-client/test/lib/containers/discussion_forum/DiscussionThread.integration.test.tsx

```diff
@@ -105,7 +105,8 @@ function renderComponent() {
   })
 }
 
-describe('Discussion Thread test', () => {
+// TODO: Fix flakiness
+describe.skip('Discussion Thread test', () => {
```

Comment:
This suite fails often. Perhaps can come back to it if/when we use these discussion forum components.